notebook/jstest.py

```diff
@@ -230,6 +230,8 @@ def __init__(self, section, xunit=True, engine='phantomjs', url=None):
         includes = '--includes=' + os.path.join(js_test_dir,'util.js')
         test_cases = os.path.join(js_test_dir, self.section)
         self.cmd = ['casperjs', 'test', includes, test_cases, '--engine=%s' % self.engine]
+        # self.cmd = ['casperjs', 'test', includes, test_cases, '--engine=%s' % self.engine,
+        #             '--remote-debugger-port=9000']
```

Comment:
Ok, I will get rid of that. Just to check if I understood correctly: we keep the `if (!mode)` case?